packages/repl/svelte.config.js

```diff
@@ -5,7 +5,4 @@ export default {
 	kit: {
 		adapter: adapter()
 	},
-	package: {
-		exports: (file) => file === 'index.js'
-	}
-};
+	};
```

Comment:
not sure if the semicolon is required? can definitely remove a couple spaces though
```suggestion
};
```